Programowanie3/Assets/Scripts/Movement.cs

```diff
@@ -5,11 +5,20 @@
 public class Movement : MonoBehaviour
 {
     [SerializeField] private float moveSpeed = 5;
+    private float startingSpeed;
+    private float temp = 0;
     private Rigidbody rb;
+    [SerializeField] private float resetTime = 5;
     // Start is called before the first frame update
     void Start()
     {
         rb = GetComponent<Rigidbody>();
+        startingSpeed = moveSpeed;
+    }
+
+    public void Return()
```

Comment:
```suggestion
    public void ResetSpeed()
```
Bardziej opisowa nazwa